src/components/ProjectSettingsTabs/VirtualWorkstationTab/CommandRow.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "@emotion/styled";
+import { SpruceFormProps } from "components/SpruceForm";
+import { size } from "constants/tokens";
+
+export const CommandRow: SpruceFormProps["ObjectFieldTemplate"] = ({
+  properties,
+}) => {
+  const [command, directory] = properties;
+
+  return (
+    <RowContainer data-cy="command-row">
+      <LeftColumn>{command.content}</LeftColumn>
+      <div>{directory.content}</div>
+    </RowContainer>
+  );
+};
+
+const LeftColumn = styled.div`
+  padding-right: ${size.s};
+`;
+
+const RowContainer = styled.div`
+  display: flex;
+
+  > div {
```

Comment:
Wouldn't this style apply to both divs anyways im curious why we don't apply it to them directly instead?